lib/uksched/include/uk/tcb_impl.h

```diff
@@ -79,7 +79,7 @@ int uk_thread_uktcb_init(struct uk_thread *thread, void *tcb);
  * @param tcb Reference to reserved space to custom thread control block
  *            that should be uninitialized.
  */
-void uk_thread_uktcb_fini(struct uk_thread *thread, void *tcb);
+void uk_thread_uktcb_fini(struct uk_thread *thread, void *tcb) __nonnull;
```

Comment:
The function already uses `UK_ASSERT`, using `__nonnull` is redundant here and would generate avoidable warnings